avocado/plugins/list.py

```diff
@@ -146,6 +151,19 @@ def configure(self, parser):
                             'Current: %(default)s')
         loader.add_loader_options(parser)
 
+        filtering = parser.add_argument_group('filtering parameters')
```

Comment:
I'll count the cost... I also don't like duplicated code, but also don't like non-explicity stuff to work around it.